Finish pr600: Added LEM-LTSR-NP footprint

poeschlr opened the pull request:
Footprint to be used with LEM-LTSR6-NP, LEM-LTSR15-NP and LEM-LTSR25-NP.
Symbol pull request https://github.com/KiCad/kicad-symbols/pull/615

This fixes the remaining points of #600.

https://www.lem.com/sites/default/files/products_datasheets/ltsr_6-np.pdf
https://www.lem.com/sites/default/files/products_datasheets/ltsr_15-np.pdf
https://www.lem.com/sites/default/files/products_datasheets/ltsr_25-np.pdf

![screenshot from 2018-08-27 23-00-09](https://user-images.githubusercontent.com/18327350/44685971-06515f80-aa4d-11e8-8aa5-d5c138f4cb38.png)

![screenshot from 2018-08-27 22-47-55](https://user-images.githubusercontent.com/18327350/44685985-179a6c00-aa4d-11e8-90fb-0bbb7a14ae04.png)

evanshultz commented:
Thanks!

From https://github.com/KiCad/kicad-footprints/pull/600#issuecomment-408253184:
- Could you simplify the description as I mentioned?
- Unrotate pads?
- I made some pad size suggestions which match our previous decisions.
- I assume you copied body dimensions including the corner radius? 3D model check?
- Some dimensions could be rounded to X.XXmm.
- Break up the horizontal silk line just above pins 4-6 where it's under the body?
- Off-grid courtyard lines (the overlapping silk Travis also mentions looks to be bogus).

poeschlr commented:
I changed the pad orientation and pad sizes. (I did not reduce it to the full extend that you suggested as it is easier to make the courtyard with slightly larger pads.)
I also fixed the drill for the smaller pads. I however went with the suggestion of the datasheet (0.8mm drill size)
I also shortened the description.
I don't know why there where off grid lines but i fixed them using a text editor. (And checked the result back in the footprint editor)

Even thought this is a through hole component and we allow copper below such parts i removed the silk line as you suggested.

The radius of the 3d model is ~1mm. The radius of the footprint is 1mm.

poeschlr commented:
Done
